app/views/users/edit_profile.html.erb

```diff
@@ -44,6 +44,24 @@
   <p>Note: Links are not shown publicly until you have earned the Participate Everywhere ability.</p>
   <div class="post-preview"></div>
 
+  <div>
+    <p>External links (your web site, blog, social media, GitHub, etc)</p>
```

Comment:
Is it worth mentioning that only 3 will be displayed?

Reply:
I think we're only going to give you three label/url textbox pairs to edit.  (Oleg suggested prefilling these rows for all users so that we don't have to figure out how to get new ones in there in the view code.)  There's no "add site" button.